apps/web/src/components/FeedItem/FeedItem.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Icon, { iconColor } from '../Icon/Icon';
+import './FeedItem.scss';
+
+export interface Props {
+  svg: React.ReactElement;
+  label: string;
+  href: string;
+}
+
+/**
+ * Renders a FeedItem component
+ *
+ * @param {Object} props - The props object.
+ * @param {React.ReactElement} props.svg - The SVG icon to display
+ * @param {string} props.label - The text to show in the label element.
+ * @param {string} props.href - the URL to use in the anchor element.
+ * @returns {JSX.Element} A JSX element
+ */
+const FeedItem = ({ svg, label, href }: Props) => {
+  return (
+    <a href={href}>
```

Comment:
We can do a little more for `a11y` here by adding an `aria-label` to our link.
```suggestion
    <a href={href} aria-label={label}>
```